tests/benchdnn/inputs/matmul/test_matmul_smoke

```diff
@@ -12,13 +12,20 @@
 --batch=shapes_2d_ci
 
 --dt=s8:s8:f32
---attr-zero-points=src:common:1+wei:common:-1+dst:common:2
+--attr-zero-points=src:common:1+wei:common:-1+dst:common:2,\
+              src:host_scalar:1+wei:host_scalar:-1+dst:host_scalar:2
 --attr-scales=src:common:0.25+wei:per_oc+dst:common:2
 --batch=shapes_2d_ci
 
 --dt=u8:s8:s32
 --attr-zero-points=
---attr-scales=src:common:0.25+wei:per_oc
+--attr-scales=src:common:0.25+wei:per_oc,\
+              src:host_scalar:0.25+wei:per_oc+dst:host_scalar:2
+--batch=shapes_2d_ci
+
+--dt=u8:s8:s32
+--attr-zero-points=
+--attr-scales=src:host_scalar:0.25+wei:per_oc
```

Comment:
Let's keep smoke test set untouched. It serves a different purpose.

Reply:
[The last sentence.](https://github.com/uxlfoundation/oneDNN/blob/main/tests/benchdnn/doc/benchdnn_input_files_naming_convention.md)